app/services/album.js

```diff
@@ -2,13 +2,26 @@ const axios = require('axios');
 const logger = require('../logger');
 const errors = require('../errors');
 
+const apiUrl = 'https://jsonplaceholder.typicode.com';
+
 exports.albums = () =>
   axios
-    .get('https://jsonplaceholder.typicode.com/albums')
+    .get(`${apiUrl}/albums`)
     .then(response => {
       logger.info(`Received albums: ${JSON.stringify(response)}`);
       return response;
     })
     .catch(() => {
-      throw errors.external_api_error('Error in external API');
+      throw errors.externalApiError('Error in external API');
```

Comment:
This is me being nitpicky, but you can add a bit more detail in the error log about what service of the external api you are requesting instead of a general 'Error in external API'. This will make it easier to figure out potential problems later on.